ui/src/holdingpen-new/containers/SearchPageContainer/SearchPageContainer.tsx

```diff
@@ -12,33 +12,55 @@ import NumberOfResults from '../../../common/components/NumberOfResults';
 import SearchPagination from '../../../common/components/SearchPagination';
 import PublicationsSelectAllContainer from '../../../authors/containers/PublicationsSelectAllContainer';
 import UnclickableTag from '../../../common/components/UnclickableTag';
-import ResultItemWithActions from '../../components/ResultItemWithActions';
-import ResultitemWithComparison from '../../components/ResultitemWithComparison';
 import AuthorResultItem from '../../components/AuthorResultItem';
+import { authToken } from '../../token';
 
 interface SearchPageContainerProps {
   data?: any;
 }
 
 const renderResultItem = (item: any) => {
-  if (item.get('id') === 123456) {
-    return <AuthorResultItem item={item} />;
-  }
-  if (item.get('id') === 6) {
-    return <ResultitemWithComparison item={item} />;
-  }
-  return <ResultItemWithActions item={item} />;
+  return <AuthorResultItem item={item} />;
 };
 
 const SearchPageContainer: React.FC<SearchPageContainerProps> = () => {
   const [loading, setLoading] = useState(true);
+  const [searchResults, setSearchResults] = useState<any[]>([]);
+  const [allSearchResults, setAllSearchResults] = useState<any[]>([]);
+  const [page, setPage] = useState(1);
+  const [size, setSize] = useState(5);
 
   const resolveLoading = () => {
     setTimeout(() => setLoading(false), 2500);
   };
 
   resolveLoading();
 
+  const getFirstPageOfSearchResults = async () => {
+    const res = await fetch(
+      `https://backoffice.dev.inspirebeta.net/api/workflows/search?page=${page}&size=${size}`,
+      authToken
+    );
+    const data = await res?.json();
+    return data.results || [];
+  };
+
+  const getAllSearchResults = async () => {
+    const res = await fetch(
+      `https://backoffice.dev.inspirebeta.net/api/workflows/search`,
```

Comment:
could we have this as a config?

Reply:
such as https://github.com/cern-sis/kubernetes/blob/master/ui/environments/inspire-prod/config.js